web/styles/dark_theme.css

```diff
@@ -709,8 +708,7 @@
         background-color: hsl(228deg 11% 17%);
     }
 
-    & thead,
-    .drafts-container .drafts-header,
+    & .drafts-container .drafts-header,
```

Comment:
Why is this line changed?

Reply:
this `thead` selector overrides the CSS from the new app variable we introduced at `app_variables.css` if we keep them both.

Reply:
OK, makes sense.